tests/mocha/block_test.js

```diff
@@ -1420,30 +1425,32 @@ suite('Blocks', function() {
       });
 
       test('removing an icon triggers a render', function() {
-        this.renderSpy.resetHistory();
         this.block.addIcon(new MockIconA());
+        this.renderSpy.resetHistory();
         this.block.removeIcon('A');
         chai.assert.isTrue(
             this.renderSpy.calledOnce,
             'Expected removing an icon to trigger a render');
       });
     });
 
-    suite.skip('Getting icons', function() {
+    suite('Getting icons', function() {
       setup(function() {
         this.block = this.workspace.newBlock('stack_block');
       });
 
-      test('all icons are returned from getIcons', function() {
-        const iconA = new MockIconA();
-        const iconB = new MockIconB();
-        this.block.addIcon(iconA);
-        this.block.addIcon(iconB);
-        chai.assert.sameMembers(
-            this.block.getIcons(),
-            [iconA, iconB],
-            'Expected getIcon to return both icons');
-      });
+      test(
+          'all icons are returned from getIcons, in order of weight',
+          function() {
+            const iconA = new MockIconA();
+            const iconB = new MockIconB();
+            this.block.addIcon(iconA);
+            this.block.addIcon(iconB);
```

Comment:
I would add these in the opposite order so that the test ensures they aren't just being returned in the order they were added.